hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2123,6 +2123,45 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator rolebinding: %w", err)
 	}
 
+	// TODO: Remove this block after initial merge of this feature. It is not needed for latest CPO version
+	if r.ManagementClusterCapabilities.Has(capabilities.CapabilityRoute) {
```

Comment:
I'm pretty sure this will break Azure HCs on AKS. There is no `openshift-ingress` and `openshift-ingress-operator` namespace in the AKS mgmt cluster.

Can we have a check for the managed services flag we pass to the HO for ARO?

Reply:
Will add a check for the version, only apply it to 4.14